patch-version-string

Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

Add the git hash from which this release is based to the end of the
existing GRUB version string.

This differentiates it from other releases that do not work.

diff --git a/configure.ac b/configure.ac
index 8888c2f..9614256 100644
--- a/configure.ac
+++ b/configure.ac
@@ -31,7 +31,7 @@ dnl (such as BUILD_CC, BUILD_CFLAGS, etc.) for the build type and variables
 dnl with the prefix "TARGET_" (such as TARGET_CC, TARGET_CFLAGS, etc.) are
 dnl used for the target type. See INSTALL for full list of variables.
 
-AC_INIT([GRUB],[2.02~beta2],[bug-grub@gnu.org])
+AC_INIT([GRUB],[2.02~beta2+e4a1fe391],[bug-grub@gnu.org])
 
 AC_CONFIG_AUX_DIR([build-aux])
 
